changelogs/fragments/35331-postgres_privs-add-all_in_schema-for-functions.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - Accept 'ALL_IN_SCHEMA' objs for 'function' type
```

Comment:
@chamini2 Sorry for late reply, 
Since changelogs are combined together in single file, this line will not related to postgre_privs. So could you please note like this -

```
- postgres_privs now accept 'ALL_IN_SCHEMA' objs for 'function' type (https://github.com/ansible/ansible/pull/35331).
```